roles/migrationcontroller/templates/mig_rbac.yml.j2

```diff
@@ -68,7 +68,11 @@ rules:
   verbs:
   - use
   resourceNames:
+{% if migration_rsync_privileged %}
```

Comment:
Unsure if this is a great idea because you'd still have to bounce pods that are using this RBAC but I like the concept of not granting privileges we don't have to.